packages/muelu/src/Graph/BrickAggregation/MueLu_BrickAggregationFactory_decl.hpp

```diff
@@ -59,18 +59,20 @@
 #include "MueLu_SingleLevelFactoryBase.hpp"
 #include "MueLu_BrickAggregationFactory_fwd.hpp"
 
+#include "MueLu_GraphBase.hpp"
+#include "MueLu_Graph_fwd.hpp"
+#include "MueLu_LWGraph_fwd.hpp"
 #include "MueLu_Level_fwd.hpp"
 #include "MueLu_Aggregates_fwd.hpp"
 #include "MueLu_Exceptions.hpp"
 #include "MueLu_Utilities_fwd.hpp"
 
 /*!
   @class BrickAggregationFactory
-  @brief Aggregation method for generating "brick" aggregates.
+  @brief Aggregation method for generating "brick" aggregates.  It also does "hotdogs" and "pancakes."
```

Comment:
mmmm, pancakes.

Reply:
German Apple Pancake.  Do it.  You will not regret.